<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiWidget</class>
 <widget class="QWidget" name="uiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniMax Algorithm</string>
  </property>
  <layout class="QGridLayout" name="mMainGridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="mTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mTabGame">
      <attribute name="title">
       <string>Tic Tac Toe</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0" colspan="2">
        <widget class="ndx::GameView" name="mGameView">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>350</height>
          </size>
         </property>
         <property name="renderHints">
          <set>QPainter::Antialiasing</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Player (X):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mScoreX">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Player (O):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mScoreO">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="mBtnGameNewRound">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>New Round</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="mBtnGameReset">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="mGameStatus">
         <property name="text">
          <string>CLICK ON NEW ROUND TO BEGIN!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="mBtnGameMiniMax">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>MiniMax</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTabWidgetVisual">
      <attribute name="title">
       <string>Visual</string>
      </attribute>
      <layout class="QGridLayout" name="mTabWidgetVisualLayout">
       <item row="0" column="0" colspan="2">
        <widget class="ndx::Controller" name="mViewController" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>View</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGraphicsView" name="mMiniMaxGraphics"/>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Data</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Depth:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="mDepth">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>4</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="mBtnGenerate">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Generate</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTabWidgetTheory">
      <attribute name="title">
       <string>Theory</string>
      </attribute>
      <layout class="QGridLayout" name="mTabWidgetTheoryLayout">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="mTextTheory">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ndx::Controller</class>
   <extends>QWidget</extends>
   <header>Controller.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ndx::GameView</class>
   <extends>QGraphicsView</extends>
   <header>GameView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
